test/Microsoft.Identity.Abstractions.Tests/AquireTokenOptionsTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Xunit;
+
+namespace Microsoft.Identity.Abstractions.Tests
+{
+    public class AquireTokenOptionsTests
+    {
+        [Fact]
+        public void ManagedIdentitySystemAssigned()
+        {
+            // App token from a system-assigned managed identity.
+            // -------------------------------------------------
+            // https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview
+            /*
+            // <managedidentity_json>
+            {
+                "AquireTokenOptions": {
+                    "ManagedIdentity": {
+                        "Source": "SystemAssigned"
```

Comment:
Personally I would be ok with:

```json
 "AquireTokenOptions": {
      "ManagedIdentity": {}
  }
```

in the case of System.Assigned managed identity. But for a beginner, what you propose is more explicit.

Reply:
I tend to agree to your proposal @jmprieur . It will also be consistent with MSAL NodeJS, which is also JSON driven.

Reply:
If we are trying to make things harder to get wrong being more explicit for beginners is the direction we should be trying to take no?